### PR TITLE
Added icons for creasoldombus custom component

### PR DESCRIPTION
**Pull request by CreasolTech**
## Type of change

- [ ] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [x] Add a new logo or icon for a custom integration (custom component)
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #
- Link to code base pull request: 
- Link to documentation pull request: 
- Link to integration documentation on our website: 

## Checklist

- [x] The added/replaced image(s) are **PNG**
- [x] Icon image size is 256x256px (`icon.png`)
- [x] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [ ] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [ ] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

File: custom_integrations/creasoldombus/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Creasol Dombus icons and logo for Home Assistant
```
Comment (frenck) [inline custom_integrations/creasoldombus/README.md:1]: This file does not belong here, please remove this file.

**Review by frenck — CHANGES_REQUESTED** [review]
The x2 version of the icon looks pretty blown up:

image

Let's remove that one, or replace it with a higher quality image.

File: custom_integrations/creasoldombus/README.md (hunk 1 above)
Comment (CreasolTech) [reply custom_integrations/creasoldombus/README.md]: Ok, modifications have been done.
Thanks a lot for the icon quality suggestion.
Paolo

**Review by frenck — APPROVED** [review]
Awesome, thanks! 👍

**Pull request merged**